kubesphere/install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Init version
+export RELEASE=v3.2.1
+
+# Install clusterconfiguration crd
+kubectl apply -f https://github.com/kubesphere/ks-installer/releases/download/$RELEASE/cluster-configuration.yaml
```

Comment:
WHy this is separate and app.yaml is separate ?
CRD is also there in app.yaml , is that different from this one ?

Reply:
@mangoGoForward

Reply:
Kubesphere supports enable the pluggable components, and the `ClusterConfiguration` in the `install.sh` is a cr to control which components should be install, it must be wait the CRD of `clusterconfigurations.installer.kubesphere.io` in the `app.yaml` applied in k8s successful.

Reply:
ok Also 
app.yaml will be applied first and then the install.sh will run. Is that fine for above ?

Reply:
Yup, and we applied a remote file in `install.sh`, should we change to apply a stdin?

Reply:
As long as you can maintain the remote file should be fine

Reply:
The remote YAML file will be updated and maintained all the time. We might only need to update the version variable in this repo.

Reply:
yeah